### PR TITLE
Test speedups

### PR DESCRIPTION
**Pull request by ernestum**
## Description

Some test parameter tweaks to reduce the linux test suite from ~10min to ~5min.
Mostly easy gains that require not in-depth changes

## Testing

Changes are in tests.

### CODE REVIEW

**Comment by ernestum** [comment]
@taufeeque9 would you mind having a look at what is up with the windows runner?

**Comment by taufeeque9** [comment]
@ernestum I reran the Windows CI tests on this branch and the master branch, and both runs succeeded. So it might have been a downstream issue that got fixed. Let me know if the Windows issue comes up again. I'd be happy to help!

**Comment by ernestum** [comment]
Thanks a lot!

**Comment by AdamGleave** [comment]
@qxcv or @Rocamonde could you review this please? Thanks!

File: src/imitation/scripts/ingredients/rl.py [hunk 1]
```diff
@@ -180,6 +189,11 @@ def load_rl_algo_from_path(
     relabel_reward_fn: Optional[RewardFn] = None,
 ) -> base_class.BaseAlgorithm:
     rl_kwargs = dict(rl_kwargs)
+
+    # TODO: this is a hack and an indicator that the rl ingredient should be refactored
+    if rl_cls == sb3.SAC:
+        del rl_kwargs["n_epochs"]
```
Comment (qxcv) [inline src/imitation/scripts/ingredients/rl.py:195]: Is it worth writing this up as an issue?

**Review by qxcv — APPROVED** [review]
LGTM (I suggested writing up the ingredient refactor as an issue, but up to you whether you do this)

File: src/imitation/scripts/ingredients/rl.py (hunk 1 above)
Comment (ernestum) [reply src/imitation/scripts/ingredients/rl.py:195]: I think it is already kind of covered by #79

**Comment by AdamGleave** [comment]
Done -- sorry for the delay

**Pull request merged**